eclipse-theia/theia
# Update Built-in Extensions Version

#### What it does

Fixes #9014

- Update necessary built-ins to version 1.45.1



#### How to test

1. `git clone https://github.com/eclipse-theia/theia`
2. `cd theia`
3. `yarn`
Confirm that all built-ins are successfully downloaded.


- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: seantan22 <sean.a.tan@ericsson.com>
